js/web/lib/wasm/jsep/util.ts

```diff
@@ -119,11 +119,11 @@ export class ShapeUtil {
     let size = 1;
     for (let i = start; i < end; i++) {
```

Comment:
If the range is empty, i.e. start === end, which is [], then this function would return 1. Is that what is expected? Don't we expected 0?

Reply:
a scalar tensor (dims.length === 0) has 1 element